themes/rr/rr.theme.sh

```diff
@@ -0,0 +1,28 @@
+#! bash oh-my-bash.module
+
+# rr is a simple one-liner prompt inspired by robbyrussell from ohmyzsh themes.
+#
+# Looks:
+#
+# ➜  anish ~ cd .bash-it/themes/dulcie
+# ➜  anish custom-dulcie git:(master ✓) # with git
+#
+# Configuration. Change these by adding them in your .bash_profile
+
+function _omb_theme_PROMPT_COMMAND() {
+    local user_name="${_omb_prompt_white}\u${_omb_prompt_reset_color}"
+    local base_directory="${_omb_prompt_bold_blue}\W${_omb_prompt_reset_color}"
+    local SCM_THEME_PROMPT_PREFIX="${_omb_prompt_bold_purple}git:(${_omb_prompt_reset_color}"
+    local SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_bold_purple})${_omb_prompt_reset_color}"
+    local SCM_THEME_PROMPT_CLEAN="${_omb_prompt_bold_green} ✓${_omb_prompt_reset_color}"
+    local SCM_THEME_PROMPT_DIRTY="${_omb_prompt_bold_red} ✗${_omb_prompt_reset_color}"
+    local arrow="${_omb_prompt_bold_purple}➜${_omb_prompt_reset_color}"
+
+    PS1="${arrow}  ${user_name} ${base_directory} "
+
+    local scm_info=$(scm_prompt_info)
+
+    PS1+=${scm_info:+$scm_info }
+}
+
+_omb_util_add_prompt_command _omb_theme_PROMPT_COMMAND
```

Comment:
Thanks for updating!

An additional point is that we need a newline character at the end of the file. Missing the newline character may cause problems with other command line tools that handle this file since the line needs to be always terminated by a newline according to the POSIX standard. I think this doesn't need to be discussed, so I'll just fix it.